frame/election-provider-multi-phase/Cargo.toml

```diff
@@ -20,7 +20,8 @@ log = { version = "0.4.14", default-features = false }
 frame-support = { version = "3.0.0", default-features = false, path = "../support" }
 frame-system = { version = "3.0.0", default-features = false, path = "../system" }
 
-sp-io ={ version = "3.0.0", default-features = false, path = "../../primitives/io" }
+sp-core = { version = "3.0.0", default-features = false, path = "../../primitives/core" }
```

Comment:
I did not find a use case of sp-core in my first skim. sure it is needed?

Reply:
It definitely was at some point, but it appears not to be anymore. Good spot! 0edfca58e